docs/best-practices/code-reviews/readme.md

```diff
@@ -146,214 +183,256 @@ Further paragraphs come after blank lines.
-  reasonable amount of time? Was the flow sane and names easy to
-  follow? Were you able to keep track through multiple files or
-  functions? Were you put off by inconsistent naming?
+  reasonable amount of time? Was the flow sane and were variable and methods names
+  easy to follow? Were you able to keep track through multiple files or functions?
+  Were you put off by inconsistent naming?
 
 - **Does the code adhere to coding guidelines and code style?** Is the
   code consistent with the project in terms of style, API conventions,
-  etc?
+  etc.? As mentioned above, we prefer to settle style debates with automated
+  tooling.
 
 - **Does this code have TODOs?** TODOs just pile up in code, and become
-  stale over time. Have the author submit a ticket on Github Issues and
-  attach the issue number to the TODO. Similarly, the CR should not have
-  large amounts of commented-out code.
+  stale over time. Have the author submit a ticket on GitHub Issues or JIRA and
+  attach the issue number to the TODO. The CR should not contain
+  commented-out code.
 
 ### Maintainability
 
 - **Read the tests.** If there are no tests and there should be, ask the
-  author to write some. Untestable features are rare, while untestable
-  implementations of features are fairly common. Check the tests
+  author to write some. Truly untestable features are rare, while untested
+  implementations of features are unfortunately common. Check the tests
   themselves: are they covering interesting cases? Are they readable?
   Does the CR lower overall test coverage? Think of ways this code
   could break. Style standards for tests are often different than core
   code, but still important.
 
-- **Does this CR have the possibility of breaking test code - Bouncers
+- **Does this CR have the possibility of breaking test code, staging stacks,
   or integrations tests?** These are often not checked as part of the
-  commit/merge checks, but having them go down is painful
+  pre-commit/merge checks, but having them go down is painful
   for everyone. Specific things to look for are: removal of test
   utilities or modes, changes in configuration, and changes in
   artifact layout/structure.
 
 - **Does this change break backward compatibility?** If so, is it OK to
-  merge the CR at this point?
+  merge the change at this point or should it be pushed into a later release?
+  Breaks can include database or schema changes, public API changes, 
+  user workflow changes, etc.
 
 - **Does this code need integration tests?** Sometimes, code can't
-  adequately be tested with unit tests alone - especially if the code
-  needs to interact with outside systems. In this case, make sure that
-  the coder either has integration tests in place or a bug filed to
-  add them.
+  be adequately tested with unit tests alone, especially if the code
+  interacts with outside systems or configuration. 
 
 - **Leave feedback on documentation, comments, and commit messages.**
```

Comment:
since you mention "external documentation" below, consider qualifying this: "Leave feedback on internal documentation, comments, ..."

Reply:
done